### PR TITLE
Fix version tags for samples

### PR DESCRIPTION
**Pull request by srnagar**
Fixes missing version tags in samples directory.

### CODE REVIEW

File: samples/azure-samples-graalvm-spring-storageexplorer/pom.xml [hunk 1]
```diff
@@ -47,14 +45,14 @@
...
-      <version>5.11.0</version>
+      <version>5.6.0</version> <!-- {x-version-update;net.java.dev.jna:jna;dependency} -->
     </dependency>
 
     <!-- this enables the Spring support for native-image compilation -->
     <dependency>
       <groupId>org.springframework.experimental</groupId>
       <artifactId>spring-native</artifactId>
-      <version>${spring-native.version}</version>
+      <version>0.11.4</version> <!-- {x-version-update;org.springframework.experimental:spring-aot-maven-plugin;external_dependency} -->
```
Comment (JonathanGiles) [inline samples/azure-samples-graalvm-spring-storageexplorer/pom.xml:55]: Should this be spring-native?

**Review by JonathanGiles — APPROVED** [review]

**Comment by JimSuplizio** [comment]
The internal spring pipeline is running against this refs/merge, when that run's [build/analyze](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2638561&view=results) completes this can be checked in and will probably have to override check enforcer to do so.

**Review by JimSuplizio — APPROVED** [review]
I'm pre-approving the changes. We'd discussed this over teams and this is the right change.

**Comment by srnagar** [comment]
The [build](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2638561&view=results) completed successfully.

**Pull request merged**